src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -14,6 +14,7 @@
       "description": "The set of runs contained in this log file.",
       "type": "array",
       "minItems": 1,
+      "uniqueItems": true,
```

Comment:
What does a log with two completely empty runs tell anybody? I think the schema _should_ insist, and the SDK enforce, that there is some way to tell one run from another. startTime/endTime would be a fine way to do it.